Geometry_Engine/Modify/FilletPolyline.cs

```diff
@@ -0,0 +1,164 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2025, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.Engine.Geometry;
+using BH.oM.Base.Attributes;
+using BH.oM.Geometry;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Net.NetworkInformation;
+
+namespace BH.Engine.Geometry
+{
+    public static partial class Modify
+    {
+        /***************************************************/
+        /**** Public Methods - Curves                  ****/
+        /***************************************************/
+
+        [Description("Returns a PolyCurve representing the input Polyline with fillets of the given radius at each internal vertex.")]
+        [Input("polyline", "The Polyline to fillet.")]
+        [Input("radius", "The radius of the fillet arc at each corner.")]
```

Comment:
```suggestion
        [Input("radius", "The radius of the fillet arc at each corner.")]
```
What defines a corner? Does this need an angular input? I assume this initially is for `pi/2` but that's not stated anywhere.

Reply:
A corner is just the vertex between two straight joined segments. This is why the input is a `Polyline` and not a `Polycurve`.

Reply:
So this creates a fillet at each vertex?

I would update the description to use consistent language of vertex rather than corner.

Reply:
Corrected term "corner" to "vertex" to be more clear